packages/mock-account-provider/app/routes/webhooks.ts

```diff
@@ -1,13 +1,13 @@
 import type { ActionArgs } from '@remix-run/node'
 import { json } from '@remix-run/node'
 import { mockAccounts } from '~/lib/accounts.server'
-import assert from 'assert'
```

Comment:
IMO `assert` should be reserved for tests only. Converted to using `throw json` as per [discussion](https://github.com/interledger/rafiki/pull/660#discussion_r989095332) with @wilsonianb

Reply:
Want to open a (good first) issue to audit usages of `assert` outside of tests?

Reply:
Yep 👍

Reply:
Issue: https://github.com/interledger/rafiki/issues/661